### ISSUE TITLE
system crash while autosaving will corrupt ALL saves

### ISSUE THREAD

**Issue by MaudDibb** [tobspr/shapez.io#1388]
thats a bit of a harsh thing when you lose all your progress and all other works from seperate sessions, all in one go. because someone trips on a power cord. how bout saving the files locally as well as on the cloud, in seperate files, so the autosave doesnt trash all your work? i know theres an export function, but seriously? autosave has to affect all saves in one go?

**Comment by EmeraldBlock** [comment]
shapez does do those things, and it usually only affects the save you're in. what does your savegames folder look like? maybe the savegames list file got corrupted

**Comment by tobspr** [comment]
Yeah I highly suspect only the savegames list file got corrupted. Try importing the savegames from your saves/ folder directly. Also there should be a ".backup" file of every file as well, which you can try using

**Comment by tneo** [comment]
The back-up file gets corrupted as well. As the back-up file has the near same timestamp as the main save file. It would be beneficial to have multiple back-ups (up to 4) as the frequency of saving is high (default every 2 minutes). A crash during save should not corrupt a back-up file. 

Importing the back-up will not help as it is corrupt as well. The only way to work-around the issue is by actively saving manually and copying the file to another location.

**Comment by tutacat** [comment]
> The back-up file gets corrupted as well. As the back-up file has the near same timestamp as the main save file

It has to be made at the same time because it's a copy of the previous version
